Rdmp.UI/PipelineUIs/DemandsInitializationUIs/ArgumentValueControls/ArgumentValueSqlUI.cs

```diff
@@ -44,6 +44,7 @@ public void SetUp(IActivateItems activator, ArgumentValueUIArgs args)
             _args = args;
 
             tbSql.Text = FormatSqlForTextbox(args.InitialValue);
+            lblSqlClause.Text = args.ContextText?.ToUpper();
```

Comment:
Why call `ToUpper`? that would be unexpected to me as a developer implementing `DemandsInitialization` and providing a lowercase or mixed case string.